### DISCUSSION TITLE
Wipe pastebin without popup

### DISCUSSION THREAD

**Discussion by plhrtr** [kRHYME7/Hyde-cli#89] [Q&A] [answered]
Hi there, 
I'm trying  to auto wipe my clipboard on a regular bases. My easiest solution would be to run `Hyde show pastebin --wipe` on system startup but then there is the confirmation pop up which is a little bit annoying to accept each time. 

Is there a way to skip this pop up? Or is there another way to wipe the pastebin? 

Thanks in advance!

**Comment by kRHYME7** [comment, marked as answer]
Hi please use the ` cliphist ` command.

**Reply by plhrtr** [reply]
Pretty easy solution 😅
Thanks!
